apps/rush/src/taskRunner/TaskRunner.ts

```diff
@@ -101,6 +101,8 @@ export default class TaskRunner {
     this._currentActiveTasks = 0;
     console.log(`Executing a maximum of ${this._parallelism} simultaneous processes...${os.EOL}`);
 
+    this._checkForCyclicDependencies([...this._tasks.values()], []);
```

Comment:
>...this._tasks.values() [](start = 38, length = 23)

Why do you need the spread operator here?

Reply:
Because values() is not an array

---
In reply to: [143555151](https://github.com/Microsoft/web-build-tools/pull/390#discussion_r143555151) [](ancestors = 143555151)